### PR TITLE
fix(discoverability): add cancel input case to FSM

### PR DESCRIPTION
**Pull request by ChenCodes**
This change accompanies the change in [box-annotations](https://github.com/box/box-annotations/pull/602)

### CODE REVIEW

File: src/lib/AnnotationControlsFSM.ts [hunk 1]
```diff
@@ -63,6 +63,9 @@ export default class AnnotationControlsFSM {
                 if (input === AnnotationInput.CREATE || input === AnnotationInput.STARTED) {
                     this.currentState = modeTempStateMap[mode] || AnnotationState.NONE;
                 }
+                if (input === AnnotationInput.CANCEL) {
+                    this.currentState = AnnotationState.NONE;
```
Comment (mxiao6) [inline src/lib/AnnotationControlsFSM.ts:67]: Is this change needed? When we're in `NONE` state, if we don't do anything, we remain in `NONE` state, right?

Comment (ChenCodes) [reply src/lib/AnnotationControlsFSM.ts:67]: Yes that's correct, this change isn't needed.
